### ISSUE TITLE
User Activation code remains in url.

### ISSUE THREAD

**Issue by bkrajendra** [octobercms/october#2657]
##### Expected behavior
**User activation process:**
Activation code/url sent to user,
User clicks on activation Link,
Users get activated and gets redirected to account page.
Gets a flash msg "Activation complete"
The activation code in (:code parameter) should be removed 

##### Actual behavior
**User activation process:**
Activation code/url sent to user,
User clicks on activation Link,
Users get activated and gets redirected to account page.
Gets a flash msg "Activation complete"
But the code in address remains as it is
If user clicks on any other page this code remains as it is in address bar,
If again clicked on account page from normal navigation menu, error is shown about user already exists.
There is no way to get rid of this activation code. This is really annoying for a non technical user.

##### Reproduce steps
Self explanatory. Just create new user and click on activation link.
##### October build
Any build from 1st to latest, this has not been fixed.
Or please let me know if there is any way to do it a standard way.

**Comment by daftspunk** [comment]
Just curious, why should it be removed?

**Comment by LukeTowers** [comment]
Closing as it has been over a month since any activity on this occurred.

**Comment by bkrajendra** [comment]
Hi all
This issue is still there. To reproduce this just try to use email activation feature for creating new user.
Or is there any other way to implementation of it, as I'm getting this issue and its really annoying.

**Comment by LukeTowers** [comment]
The support demand for October is increasing, while the available time the founders have to give remains the same. These issues will be handled in time, however there are ways you can help:

- Submit a pull request that introduces this feature and wait for it to be reviewed
- [Make a financial contribution to the project](http://octobercms.com/fundraising) so we can allocate more time towards making improvements like these that benefit everyone

Basically, if you want it fixed right now then do some debugging and provide a PR that we can review. Thanks!

**Comment by bkrajendra** [comment]
Ohkk! Got it. I'll do it. 
I was not knowing about this.
I just commented back as the issue is flagged abandoned while it's still like a bug.
I'll look into it and see if I can resolve it myself.
Thanks for your time!
